components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -176,8 +181,12 @@ class SitePermissionsFeature(
             return null
         }
 
-        val permissionFromStorage = withContext(ioCoroutineScope.coroutineContext) {
-            storage.findSitePermissionsBy(request.host)
+        val permissionFromStorage = if (session.private) {
+            null
+        } else {
```

Comment:
Is the opposite really true too? On desktop permissions granted in normal browsing seem to be granted in private browsing automatically too.

Reply:
You right, I will remove this one!